src/language-html/printer-html.js

```diff
@@ -113,7 +113,7 @@ function embed(path, print, textToDoc, options) {
 
       // lit-html: html`<my-element obj=${obj}></my-element>`
       if (
-        /^PRETTIER_HTML_PLACEHOLDER_\d+_IN_JS$/.test(
+        /^PRETTIER_PLACEHOLDER_\d+_IN_JS$/.test(
```

Comment:
I think it is not good avoiding `HTML`, for `css` we also use `CSS`

Reply:
Ok, I put it back.

Reply:
However, these placeholders are long, which causes unneeded line breaks. Can we make them shorter?